node/basic-cloud-functions-bot/index.js

```diff
@@ -18,190 +18,196 @@
+ * @param {String} actionMethodName a simple string that can be passed to choose
+ * what actionMethodName is to be specified
  * @return {Object} a widget with an image and button
  */
-function getImageButtonWidget(imageUrl, actionMethodName = "choose") {
+function getImageButtonWidget(imageUrl, actionMethodName = 'choose') {
   return {
     image: {
-      imageUrl
+      imageUrl,
     },
     buttons: [{
         textButton: {
-          text: "Select",
+          text: 'Select',
           onClick: {
             action: {
               actionMethodName,
               parameters: [{
-                  key: "url",
-                  value: imageUrl
-                }]
-            }
-          }
-        }
-      }]
+                  key: 'url',
+                  value: imageUrl,
+                }],
+            },
+          },
+        },
+      }],
   };
 }
 
 /**
- * Returns a Single Card with a Single Image
+ * Returns a Single Card with a Single Image.
  * @param {String} imageUrl
- * @returns {Object}
+ * @return {Object}
  */
 function getSingleImage(imageUrl) {
   return {
     cards: [{
         sections: [{
             widgets: [{
                 image: {
-                  imageUrl
-                }
-              }]
-          }]
-      }]
+                  imageUrl,
+                },
+              }],
+          }],
+      }],
   };
 }
 
 /**
- * Returns a Card List Commands/Usage of the bot
- * @returns {Object}
+ * Returns a Card List Commands/Usage of the bot.
+ * @return {Object}
  */
-function getHelp({ user }) {
+function getHelp({user}) {
   let commands = [
-    ["Your First Command", `@${BOTNAME} {command name}`],
-    ["Interactive Command", `@${BOTNAME} i love dogs`]
+    ['Your First Command', `@${BOTNAME} {command name}`],
+    ['Interactive Command', `@${BOTNAME} i love dogs`],
   ];
   return {
     cards: [{
         header: {
           title: `Need Help ${user.displayName}?`,
-          subtitle: "Examples Below:",
+          subtitle: 'Examples Below:',
           imageUrl:
-            "https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Google_%22G%22_Logo.svg/2000px-Google_%22G%22_Logo.svg.png",
-          imageStyle: "IMAGE"
+            'https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Google_%22G%22_Logo.svg/2000px-Google_%22G%22_Logo.svg.png',
+          imageStyle: 'IMAGE',
         },
         sections: [{
-            widgets: getWidgetValuePair(commands)
-          }]
-      }]
+            widgets: getWidgetValuePair(commands),
+          }],
+      }],
   };
 }
 
 /**
- * Creates an array of widget objects with key value pairs
- * @param {Array} commands an array of array of length 2 (aka array of tuples or key value pairs)
- * @returns {Array}
+ * Creates an array of widget objects with key value pairs.
+ * @param {Array} commands an array of array of length 2 (aka array of tuples or
```

Comment:
Can we please have a better type than `Array`?
`object[]` would be better (param can't be [2, 3]).

Reply:
Not looking to improve javadoc in this PR.